app/src/main/res/values/strings.xml

```diff
@@ -796,7 +796,7 @@
     <string name="hide_guide_text">Hide Guide</string>
 
     <string name="wave_gen_guide_intro">The wave generator can be used to generate different types of waves like
-        Sine wave, Square wave, Saw Tooth wave and allow us to changes their characteristics like frequency, phase, duty.
+        Sine wave, Square wave, Saw Tooth wave and allows us to change their characteristics like frequency, phase, duty.
```

Comment:
This is a simple mistake try to fix some big issue.

Reply:
Ok, thank you, sir, for the review.